Cabal/src/Distribution/Simple/Command.hs

```diff
@@ -49,6 +49,9 @@ module Distribution.Simple.Command (
   -- ** Running commands
   commandsRun,
 
+-- * Calculating edit distance
```

Comment:
I wonder if this is better exported from `Distribution.Simple.Utils`.

Reply:
I agree! I was meant to do that and  in the mean time forgot, thanks for the flag.